# Bioinformatics Tutorials and Workflows

ğŸ“š A repository of repositories containing bioinformatics tutorials.

View as a [web page](https://rngoodman.github.io/tutorials/) or [Github repository](https://github.com/rngoodman/tutorials).

## R programming

### ğŸƒâ€â™‚ [R boot camp](https://github.com/rngoodman/R-boot-camp)

A look at the basics of the R programming language to get you up and running on R. This bootcamp covers the following the installation of R & R Studio, the basics of R and how to read in data and inspect its contents.

### ğŸ“ [R markdown](https://github.com/rngoodman/R-Markdown-Basics)

A guide to the basics of R markdown and how to upload a project onto Github from R studio.

## Comparative genomics

### ğŸ”˜ [Circular genome comparison](https://github.com/rngoodman/circular-genome-comparisons)
Visualisation of BLAST genome alignments with circos plots in R. 

## Metagenomics & microbiome analysis

### ğŸ¦  [Clustering Human Microbiome Project (HMP) 16S rRNA data](https://github.com/rngoodman/clustering-HMP-data)
This workflow uses the Phyloseq package in R to cluster 16S rRNA data from the human microbiome project (HMP) and determine if it corresponds to a body site, such as saliva, stool, or sebum. The HMPv35 dataset, sequenced from amplicons of the rRNA 16S variable regions 3-5, is used.

## Co-occurrence analysis

### â†”ï¸ [Co-occurence analysis of AMR genes](https://github.com/rngoodman/co-occurrence-analysis)

Co-occurrence analysis of AMR genes highlights the genetic location of AMR genes relative to each other across genomes. Since short read sequencing makesÂ de novoÂ genome assembly difficult at the site of repeated sequences, which cannot be bridged if the reads are shorter than the repeat length, we used co-occurrence analysis to overcome these limitations of short-read sequencing. Co-occurrence analysis allows for the comparison of genetic interactions across genomes. It can show which genes are carried together hinting at carriage on the same or different replicons. We ran a probabilistic analysis of gene co-occurrence to understand the genetic location of AMR genes relative to each other in short-read genomic data. 



