# Bioinformatics Tutorials and Workflows

ğŸ“š A repository of repositories containing bioinformatics tutorials.

View as a [web page](https://rngoodman.github.io/tutorials/) or [Github repository](https://github.com/rngoodman/tutorials).

## R programming

### ğŸƒâ€â™‚ [R boot camp](https://github.com/rngoodman/R-boot-camp)

A look at the basics of the R programming language to get you up and running on R. This bootcamp covers the following the installation of R & R Studio, the basics of R and how to read in data and inspect its contents.

### ğŸ“ [R markdown](https://github.com/rngoodman/R-Markdown-Basics)

A guide to the basics of R markdown and how to upload a project onto Github from R studio.

## Comparative genomics

### ğŸ”˜ [Circular genome comparison](https://github.com/rngoodman/circular-genome-comparisons)
Visualisation of BLAST genome alignments with circos plots in R.

## Metagenomics & microbiome analysis

### ğŸ¦  [Clustering Human Microbiome Project (HMP) 16S rRNA data](https://github.com/rngoodman/clustering-HMP-data)
This workflow uses the Phyloseq package in R to cluster 16S rRNA data from the human microbiome project (HMP) and determine if it corresponds to a body site, such as saliva, stool, or sebum. The HMPv35 dataset is used which was sequenced from amplicons of the rRNA 16S variable regions 3-5.

## Co-occurrence analysis

### â†”ï¸ [Co-occurence analysis of AMR genes](https://github.com/rngoodman/co-occurrence-analysis)

A guide to co-occurrence analysis of AMR genes which highlights the genetic location of AMR genes relative to each other across genomes with short-read genomic data. 

## Workflows

### ğŸ§¬ [ShortSeeker](https://github.com/rngoodman/ShortSeeker)

An automated short-read bacterial genome assembly and analysis tool implemented in Snakemake. It takes paired-end gzipped fastq short-read files. It assembles and analyses the genomes producing a html report with MLST sequence typing and acquired AMR genes for multiple genomes.






